src/cmdlib.sh

```diff
@@ -288,6 +287,47 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+    mkdir -p "${workdir}"/tmp/buildinfo
+    rpm-ostree compose tree --repo="${tmprepo}" --print-only "${manifest}" > "${manifest_tmp_json}"
+    base_repos=$(jq .repos "${manifest_tmp_json}")
+    rm -f "${manifest_tmp_json}"
+
+    # Get the data form content_sets.yaml and map the repo names given in '$base_repos' to their corresponding
+    # pulp repository IDs provided in https://www.redhat.com/security/data/metrics/repository-to-cpe.json
+    python3 -c "
+import json, yaml;
+# Open the yaml and load the data
+f = open('$1')
```

Comment:
Instead of using `$1` and `$2` like this, let's declare some `local` variables upfront at the beginning of the function with clearer names.